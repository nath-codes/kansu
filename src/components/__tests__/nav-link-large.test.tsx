import 'jest-styled-components'
import React from 'react'

import NavLinkLarge from '../nav-link-large'
import { renderWithTheme } from './helpers'

describe('NavLinkLarge', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(<NavLinkLarge title="foo" url="#" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
