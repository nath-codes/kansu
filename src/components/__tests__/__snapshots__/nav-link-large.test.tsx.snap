// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLinkLarge renders correctly 1`] = `
.c0 {
  color: #49555e;
  display: block;
  font-family: "Roboto",sans-serif;
  font-weight: bold;
  font-size: 1rem;
  -webkit-letter-spacing: 2.3px;
  -moz-letter-spacing: 2.3px;
  -ms-letter-spacing: 2.3px;
  letter-spacing: 2.3px;
  position: relative;
  text-transform: uppercase;
  -webkit-transition: color 0.3s;
  transition: color 0.3s;
}

.c0:hover {
  color: #ffffff;
}

.c0.active {
  color: #ffffff;
  position: relative;
}

.c0.active:after {
  border-bottom: solid 0.375rem #ffffff;
  bottom: -0.375rem;
  content: '';
  left: 0;
  position: absolute;
  right: 0;
}

@media only screen and (min-width:980px) {
  .c0.active:after {
    bottom: -0.75rem;
  }
}

<a
  activeClassName="active"
  aria-label="foo page"
  className="c0"
  href="#"
>
  foo
</a>
`;
